docs/reference/manifest/customtab.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: CustomTab element in the manifest file
 description: On the ribbon, you specify which tab and group for their add-in commands.
-ms.date: 01/24/2020
+ms.date: 11/01/2020
 localization_priority: Normal
 ---
 
 # CustomTab element
 
-On the ribbon, you specify which tab and group for their add-in commands. This can either be on the default tab (either **Home**, **Message**, or **Meeting**), or on a custom tab defined by the add-in.
+On the ribbon, you specify the tab and group for your add-in commands. This can either be on the default tab (either **Home**, **Message**, or **Meeting**), or on a custom tab defined by the add-in.
```

Comment:
```suggestion
On the ribbon, specify the tab and group for your add-in commands. This can either be on the default tab (either **Home**, **Message**, or **Meeting**), or on a custom tab defined by the add-in.
```